src/components/interview.ts

```diff
@@ -3,14 +3,34 @@ import Discord from 'discord.js';
 import _ from 'lodash';
 
 const RESULTS_PER_PAGE = 6;
-
+const available_domains: { [key: string]: string } = {
+  frontend: 'Frontend',
+  backend: 'Backend',
+  design: 'Design',
+  pm: 'PM'
+};
 interface Interviewer {
-  user_id: number;
+  user_id: string;
   link: string;
 }
 
+function domainString() {
+  let output = '';
+  let first_domain = true;
+  for (const i of Object.values(available_domains)) {
```

Comment:
maybe using `join` would help make this cleaner https://lodash.com/docs/4.17.15#join